src/components/tutorial-collection-cards/components/card-badges/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { IconPlay16 } from '@hashicorp/flight-icons/svg-react/play-16'
+import { IconTerminalScreen16 } from '@hashicorp/flight-icons/svg-react/terminal-screen-16'
+import Badge from 'components/badge'
+import ProductIcon from 'components/product-icon'
+import { ProductSlug } from 'types/products'
+import { CardBadgesProps, CardBadgeOption } from './types'
+import s from './card-badges.module.css'
+
+/**
+ * Potential optimization: construct this map
+ * from productSlugs in `lib/products`, eg:
+ *
+ * function buildProductIconMap(): Record<ProductSlug, JSX.Element> {
+ *   const icons = {}
+ *   for (const slug of productSlugs) {
+ *     icons[slug] = <ProductIcon productSlug={slug} />
+ *   }
+ *   return icons
+ * }
+ *
+ * Deferred this as I couldn't get it to work as expected TypeScript.
+ */
+const PRODUCT_ICON_MAP: Record<ProductSlug, JSX.Element> = {
+  boundary: <ProductIcon productSlug="boundary" />,
+  consul: <ProductIcon productSlug="consul" />,
+  nomad: <ProductIcon productSlug="nomad" />,
+  packer: <ProductIcon productSlug="packer" />,
+  sentinel: <ProductIcon productSlug="sentinel" />, // note: is null!
+  terraform: <ProductIcon productSlug="terraform" />,
+  vault: <ProductIcon productSlug="vault" />,
+  vagrant: <ProductIcon productSlug="vagrant" />,
+  waypoint: <ProductIcon productSlug="waypoint" />,
+  hcp: <ProductIcon productSlug="hcp" />,
```

Comment:
In learn we call this `cloud`, just to be clear as what's coming in the data, which references `ProductOption`. Also sentinel isn't a valid learn product so I don't think we need to include that. 
```suggestion
  cloud: <ProductIcon productSlug="hcp" />,
```

Reply:
💭 Also the `ProductOption` from learn client types is an enum so it could help you generate this more easily?

Reply:
Ah wait so I realize here that i need to _drop_ `cloud` or `hcp` right? Since `cloud` isn't a possible option? (Much like `sentinel`?)

Reply:
✅ I've updated to remove the `hcp` key (so no `cloud` key either). Lemme know if I should do something different 🙇‍♂️